### DISCUSSION TITLE
分享一个bing代理

### DISCUSSION THREAD

**Discussion by caizeming** [PenUniverse/PenMods#246] [奇技淫巧]
[bing-cookies.json.txt](https://github.com/user-attachments/files/16266538/bing-cookies.json.txt)点击下载json文件
1.Signature申请地址为https://bing.dili204.com/turing/conversation/create？
2.ChatHub地址为wss://bing.dili204.com/sydney/chathub/ 
如果你是1.2.0版本，Signature申请地址需要多加一个参数，地址为https://bing.dili204.com/turing/conversation/create?bundleVersion=1.0
填入词典笔就可以用了，记得重载一下cookies

**Comment by caizeming** [comment]
附有json文件

**Comment by nosgk** [comment]
用的哪个项目推荐一下 想自己整一个

**Reply by caizeming** [reply]
我是找国内有人搭建好的，可以不用搭梯子的bing聊天网站，然后进行搭建的。
你用我这个也挺好用的咯，想自己搭建一个也行，你可以参照前面一个作者分享的搭建bing的方法，或者找到在国内可以打开的bing聊天网站，然后进行搭建。。 。。怎么说呢？我也不是专业的，你用我这个就挺好的啦，嘿嘿

**Comment by XiexieLihua** [comment]
无法创建会话，网络问题，服务器未返回任何数据，这是什么原因？

**Comment by xue23333** [comment]
这json格式都是乱的啊 读都读不出来

**Comment by ThisisYLaaa** [comment]
json修好了，但是服务器未返回任何数据=-=所以现在还能用吗
